testing-daml-models/daml-scripts/3_queries_and_assertions.md

```diff
@@ -0,0 +1,44 @@
+Just as in scenarios, with DAML script you can issue queries against the ledger and make assertions
+on the results.
+
+Let's query the ledger for `User` contracts from the operator point of view. Paste the following
+line before the final `debug` line.
```

Comment:
```suggestion
line before the final `debug` line. Make sure it has the same indentation as `forA_ ps $ \p -> do`.
```

Reply:
I mentioned the indentation one time. I kind of don't want to litter the text with indentation hints, I mean it's supposed to be an intermediate level tutorial, so I expect the user to have some understanding for DAML at this point.